### PR TITLE
feat: add cluster primary judge

### PR DESCRIPTION
**Pull request by sinkhaha**
cluster.fork() method only be called from the primary process. solve the issue https://github.com/eggjs/egg/issues/5061

### CODE REVIEW

**Comment by webjohnjiang** [comment]
Thank you, but i dont know why it fail only in my docker.

**Comment by fengmk2** [comment]
Can you add a test case for this change?

File: index.js [hunk 1]
```diff
@@ -21,6 +21,9 @@ var KILL_SIGNAL = 'SIGTERM';
 var reloading = false;
 var reloadPedding = false;
 function reload(count) {
+  if (cluster.isWorker) {
```
Comment (fengmk2) [inline index.js:24]: the reload action is executing on master process, cluster fork should be exists.

File: test/worker.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
 var http = require('http');
+var reload = require('../');
```
Comment (fengmk2) [inline test/worker.js:2]: Calling reload on worker process then throw error is the right way.

Comment (sinkhaha) [reply test/worker.js:2]: 是的，但在worker进程调fork报错会导致调用程序崩溃，是否判断是worker调用则打印个日志提示会好点

**Comment by sinkhaha** [comment]
> Thank you, but i dont know why it fail only in my docker.

有没有可能是哪里设置了这个环境变量 process.env.NODE_UNIQUE_ID

File: test/worker.js (hunk 2 above)
Comment (fengmk2) [reply test/worker.js:2]: @sinkhaha 嗯，可以加一个 assert 断言判断提升错误体验。

**Pull request closed without merging**